src/components/AppNavigation/CalendarList/Trashbin.vue

```diff
@@ -143,6 +155,7 @@ export default {
 					subline,
 					url: vobject.uri,
 					deletedAt: vobject.dav._props['{http://nextcloud.com/ns}deleted-at'],
+					color: vobject.calendar.color ?? uidToHexColor(vobject.calendar.displayName),
```

Comment:
I must have missed this.

It should be `vobject.color` (the event's own color), then fallback to the `vobject.calendar.color`, and finally fallback to the color generated from the calendar name.

Reply:
Ahh right, events can have colors too. I thought we always show the color of the calendar :man_facepalming: 

Many thanks for your thorough review :)